index.js

```diff
@@ -210,18 +184,16 @@ module.exports = {
       this.setGasPrice.bind(this)
     ], function (err) {
       if (err) return self.retryConnect(err, options, callback);
-      self.state.connection = {
-        http: self.rpc.nodes.local || self.rpc.nodes.hosted,
-        ws: self.rpc.wsUrl,
-        ipc: self.rpc.ipcpath
-      };
+      self.state.connection = true;
```

Comment:
My preference is to revert this change, if possible -- I found it quite useful having a `connection` object come back from `connect()` with the HTTP, WS, and/or IPC endpoints I'm connected to.